src/containers/views/Configuration.js

```diff
@@ -61,16 +72,33 @@ export class Configuration extends Component {
         {errors && errors.length > 0 && <Errors errors={errors} />}
         <div className="content-header">
           <h1>Configuration</h1>
-          <div className="page-buttons">
+          <div className="page-buttons multiple">
+            <Button // TODO: Hide toggle for non-YAML config files (e.g. '_config.toml')
+              onClick={this.toggleView.bind(this)}
```

Comment:
Since we started binding `this` in the `constructor` with `handleClickSave`, let's keep doing it for other ones as well.